app.ts

```diff
@@ -0,0 +1,372 @@
+let api: string = "http://localhost:2050/";
+let isFunctionRunning = false;
+let currentPage = 1;
+let firstPage = 1
+let lastPage = firstPage + 9
+let currentFromID = 1;
+let currentToID = 20;
+let difference = 0
+
+
+// This function  will handle retrieving the records from the api
+async function getRecords(fromID: number, toID: number): Promise<Array<Array<string>>> {
+	try {
+		const data = await fetch(`${api}records?from=${fromID}&to=${toID}`);
+		const records: Array<Array<string>> = await data.json();
+		return records;
+
+	} catch (error) {
+		console.error(error);
+		throw error;
+	}
+}
+
+// This function  will handle retrieving the columns from the api
+async function getColumns(): Promise<Array<string>> {
+	try {
+		const data = await fetch(`${api}columns`);
+		const columns: Array<string> = await data.json();
```

Comment:
```suggestion
		const columns: string[]= JSON.parse(data);
```

Reply:
Always make use of `.json()` instead of `JSON.parse()` when able. But the `string[]` is still the correct comment.